boa/src/syntax/lexer/tests.rs

```diff
@@ -336,7 +336,6 @@ fn test_two_divisions_in_expression() {
     let s = "    return a !== 0 || 1 / a === 1 / b;";
     let mut lexer = Lexer::new(s);
     lexer.lex().expect("failed to lex");
-    dbg!(&lexer.tokens);
```

Comment:
This `dbg!` here was to be able to debug the token generation from the lexer when we activate this test (when we solve the goal-oriented lexer). We can remove it, but not sure if there is a need.